### PR TITLE
Update firstfile.txt

### PR DESCRIPTION
**Pull request by matslinden**
@astrids8d what...?
@matslinden now

### CODE REVIEW

File: firstfile.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 The first source code updated in topic1another
+Why not add this?
```
Comment (matslinden) [inline firstfile.txt]: Well ok....

Comment (astrids8d) [inline firstfile.txt]: Noooo, this would break everything !!

File: firstfile.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
 The first source code updated in topic1another
-Why not add this?
+Why not add that!
```
Comment (astrids8d) [inline firstfile.txt]: Ah, that fixed the problem :-)

**Comment by matslinden** [comment]
nice => :shipit:

**Pull request merged**